src/components/interactive-builder/modals/question/question-form/rendering-types/inputs/select/select-answers.component.tsx

```diff
@@ -1,11 +1,12 @@
 import React, { useCallback, useEffect, useMemo, useState } from 'react';
-import { Tag, MultiSelect, Stack } from '@carbon/react';
+import { Tag, MultiSelect, Stack, InlineNotification } from '@carbon/react';
+import { WarningAltFilled } from '@carbon/react/icons';
 import { useTranslation } from 'react-i18next';
 import ConceptSearch from '../../../common/concept-search/concept-search.component';
 import { useFormField } from '../../../../form-field-context';
+import { fetchConceptById } from '@resources/concept.resource';
 import type { Concept } from '@types';
 import styles from './select-answers.scss';
-
 interface AnswerItem {
```

Comment:
Nit: I think you've accidentally deleted a new line here
```suggestion

interface AnswerItem {
```